o8g/Scripts/actions.py

```diff
@@ -861,7 +870,7 @@ def addBlessCurse(group, isBless, who=me):
 
     #check current Tokens in Bag
     updateBlessCurse()
-    if ((cb.markers[Bless] >= 10) and isBless) or ((cb.markers[Curse] >= 10) and not isBless):
+    if ((cb.markers[Bless] + b >= 10) and isBless) or ((cb.markers[Curse] + c >= 10) and not isBless):
```

Comment:
The idea is, that the numbers on the chaosbag represent the total number of tokens in play. Therefore this shouldn't be necessary.